ova/README.md

```diff
@@ -0,0 +1,64 @@
+Wazuh
+=====
+
+[![Slack](https://img.shields.io/badge/slack-join-blue.svg)](https://wazuh.com/community/join-us-on-slack/)
+[![Email](https://img.shields.io/badge/email-join-blue.svg)](https://groups.google.com/forum/#!forum/wazuh)
+[![Documentation](https://img.shields.io/badge/docs-view-green.svg)](https://documentation.wazuh.com)
+[![Documentation](https://img.shields.io/badge/web-view-green.svg)](https://wazuh.com)
+
+Wazuh is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity monitoring, policy monitoring, rootkit detection, real-time alerting, active response, vulnerability detector, etc.
+
+In this repository, you can find the necessary tools to build your own OVA file with all Wazuh componets installed.
+
+## Tools needed to build the OVA:
+
+To build an OVA you need to install the following tools:
+- `Virtual Box`: [instalation guide](https://www.virtualbox.org/manual/UserManual.html#installation)
+- `Vagrant`: [instalation guide](https://www.vagrantup.com/docs/installation/)
+- `Git`:  [installation guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
+- `Python`
```

Comment:
There is no installation guide for Python. Please add a guide.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/162/commits/99bd73a341721442e0fc065982bd83743368562d